composer.json

```diff
@@ -13,11 +13,11 @@
     ],
     "require": {
         "php": ">=5.6.0",
-        "psr/log": "~1.0"
```

Comment:
Could we move the changes to these 3 lines to an other PR? I prefer not to mix the heavy moving of files with these "dependency" logic changes.